client/landing/stepper/utils/get-flow-from-url.ts

```diff
@@ -1,16 +1,24 @@
-import { matchPath } from 'react-router-dom';
```

Comment:
I think this is being removed because this module now has to be included on the server.

Importing from `'react-router-dom'` was actually wrong. You can import the `matchPath` from the `'react-router'` package so it can be imported on the server and the client.

The reason I like the idea of continuing to use the route matching logic from React Router is because we want it to match the logic used by the client-side stepper router. And [the stepper internals use `getFlowFromURL` too](https://github.com/Automattic/wp-calypso/blob/c12d53f98fc0adcfb35f86d0ba2592bd908b1c11/client/landing/stepper/index.tsx#L87). So it seems to keep all that logic matching.